src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.stablecoins)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/stablecoins/">
+                    What are stablecoins?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Non-fungible tokens"
+                description="Non-fungible tokens represent ownership of unique items."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.infrastructureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/nft/">What are NFTs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized autonomous organizations"
+                description="Organizations that run without the need for coordination by a central entity."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dao)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dao/">What are DAOs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized applications"
+                description="Are creating a new digital economy of peer-to-peer services."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.developersEthBlocks)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dapps/">Explore dapps</ButtonLink>
+                </>
+              </Card>
+              <CardGradient
+                title="Emerging use cases"
+                description="There are also other prominent industries being disrupted by blockchain:"
+              >
+                <ul>
+                  <li>
+                    <Link to="/decentralized-identity/">
+                      Self-sovereign identity
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="/social-networks/">
+                      Decentralized social networks
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://future.com/what-is-decentralized-science-aka-desci/">
+                      Decentralized Science (DeSci)
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://decrypt.co/resources/what-are-play-to-earn-games-how-players-are-making-a-living-with-nfts">
+                      Play-to-earn games
```

Comment:
Should we introduce the acronym here? No strong opinion from me @konopkja 

```suggestion
                      Play-to-earn games (P2E)
```